jsonschema-core.xml

```diff
@@ -800,49 +800,291 @@
+                performing unusually strict syntactical validation, as might be done
+                during a development and testing cycle.
             </t>
+            <section title='The "$schema" Keyword'>
+                <t>
+                    The "$schema" keyword is both used as a JSON Schema feature set identifier and
+                    the location of a resource which is itself a JSON Schema, which describes any
+                    schema written for this particular feature set.
+                </t>
+                <t>
+                    The value of this keyword MUST be a <xref target="RFC3986">URI</xref>
+                    (containing a scheme) and this URI MUST be normalized.
+                    The current schema MUST be valid against the meta-schema identified by this URI.
+                </t>
+                <t>
+                    If this URI identifies a retrievable resource, that resource SHOULD be of
+                    media type "application/schema+json".
+                </t>
+                <t>
+                    The "$schema" keyword SHOULD be used in a root schema.
+                    It MUST NOT appear in subschemas.
+                </t>
+                <t>
+                    <cref>
+                        Using multiple "$schema" keywords in the same document would imply that the
+                        feature set and therefore behavior can change within a document.  This would
+                        necessitate resolving a number of implementation concerns that have not yet
+                        been clearly defined.  So, while the pattern of using "$schema" only in root
+                        schemas is likely to remain the best practice for schema authoring,
+                        implementation behavior is subject to be revised or liberalized in
+                        future drafts.
+                    </cref>
+                    <!--
+                        In particular, the process of validating an instance, including validating a
+                        schema as an instance against its meta-schema, only allows for a single set
+                        of rules across the entire instance document.  There is no equivalent of
+                        changing the meta-schema partway through the validation for non-schema
+                        instances.
+                    -->
+                </t>
+                <t>
+                    Values for this property are defined elsewhere in this and other documents,
+                    and by other parties.  JSON Schema implementations SHOULD implement support
+                    for current and previous published drafts of JSON Schema vocabularies
+                    as deemed reasonable.
+                </t>
+            </section>
+            <section title='The "$vocabulary" Keyword'>
+                <t>
+                    The "$vocabulary" keyword, like all keywords in meta-schemas,
```

Comment:
Are you sure about this? My impression is that "$ keywords" in a schema relate to the schema itself, and "non-$ keywords" relate to the instance. The same for meta-schema vs schema.

Therefore, `$vocabulary` would be the first "$ keyword" relating to the instance (schema) instead of the schema (meta-schema). Design smell?

Reply:
However, `$vocabulary` has effect on the interpretation of the instance (which circumstantially for meta-schemas is an schema).

If we consider meta-schemas a different kind of animal than schemas, then according to the rule of three $-keywords have no effect on the meta-schema.

Reply:
`$schema` also has an effect on the interpretation of the instance. I don't see how this is different...

Reply:
I'll take a look at the wording.

I don't think anyone is in a disagreement over the `$` prefix.

Reply:
I reworked this and I think it's stronger.  I don't know that it flows as well as it could, but I think it is more clear, so thanks, @jgonzalezdr